tests/network/connectivity/test_pod_network.py

```diff
@@ -73,12 +74,30 @@ def cloud_init_ipv6_network_data(dual_stack_network_data):
     return compose_cloud_init_data_dict(ipv6_network_data=dual_stack_network_data)
 
 
+@pytest.mark.parametrize(
+    "ip_family",
+    [
+        pytest.param(
+            IPV4_STR,
+            marks=[
+                pytest.mark.polarion("CNV-2332"),
+                pytest.mark.ipv4,
+            ],
+        ),
+        pytest.param(
+            IPV6_STR,
+            marks=[
+                pytest.mark.polarion("CNV-11845"),
+                pytest.mark.ipv6,
+                pytest.mark.jira("CNV-58530 ", run=True),
+            ],
+        ),
+    ],
+    indirect=False,
+)
 @pytest.mark.gating
```

Comment:
ditto here, only v4 part should have gating marker

Reply:
Answered above